compose/config/fields_schema_v2.json

```diff
@@ -41,6 +41,13 @@
             "^.+$": {"type": ["string", "number"]}
           },
           "additionalProperties": false
+        },
+        "external": {
+          "type": ["boolean", "object"],
+          "properties": {
+            "name": {"type": "string"}
+          },
+          "additionalProperties": false
```

Comment:
Well, yeah, the 2 other options won't really do anything if `external` is specified. I'll see if JSON schema has a thing for that.